panoptes_client/subject.py

```diff
@@ -226,19 +239,22 @@ def subject_workflow_status(self, workflow_id):
         """
         return next(SubjectWorkflowStatus.where(subject_id=self.id, workflow_id=workflow_id))
 
-    def add_location(self, location):
+    def add_location(self, location: str, manual_mimetype: str = None):
         """
         Add a media location to this subject.
 
         - **location** can be an open :py:class:`file` object, a path to a
           local file, or a :py:class:`dict` containing MIME types and URLs for
           remote media.
 
+        - **manual_mimetype** optional, passes in a specific MIME type for media item.
+
         Examples::
 
             subject.add_location(my_file)
             subject.add_location('/data/image.jpg')
             subject.add_location({'image/png': 'https://example.com/image.png'})
+            subject.add_location(my_file, 'image/png')
```

Comment:
```suggestion
            subject.add_location(my_file, manual_mimetype='image/png')
```

Reply:
Although I'm pretty sure you can pass in the `manual_mimetype` argument positionally (i.e., without the keyword name), I think it is better to include the keyword name explicitly in the example.